OctaneVSPlugin/View/PluginWindowManager.cs

```diff
@@ -51,6 +51,9 @@ internal static DetailsToolWindow ObtainDetailsWindow(MainWindowPackage package,
         /// </summary>
         internal static SearchToolWindow ObtainSearchWindow(MainWindowPackage package)
         {
+            if (package == null)
```

Comment:
ternary operator ? it's your style choice though

Reply:
@atothhpe done